### PR TITLE
Updated NeedsViralLoadTestCalculation and NeedsViralLoadTestCalculationTest - removed LTFU

### PR DESCRIPTION
**Pull request by patryllus**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/UnSuppressedVLPatientsCalculation.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import org.openmrs.module.kenyacore.calculation.BooleanResult;
+import org.openmrs.module.kenyaemr.calculation.EmrCalculationUtils;
+import org.openmrs.module.kenyaemr.calculation.library.hiv.art.LastViralLoadCalculation;
+import org.openmrs.module.kenyaemr.calculation.library.hiv.art.LowDetectableViralLoadCalculation;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Calculates  patients are have unsuppressed VL results as last results given
```
Comment (njorocs) [inline api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/UnSuppressedVLPatientsCalculation.java]: Word the comment appropriately

File: api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/art/MoreThanThreeMonthsOnARTCalculation.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import org.openmrs.module.kenyacore.calculation.BooleanResult;
+import org.openmrs.module.kenyaemr.calculation.EmrCalculationUtils;
+import org.openmrs.module.reporting.common.Age;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * Calculates  patients are have been on ART for more than 3 months (>3)
```
Comment (njorocs) [inline api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/art/MoreThanThreeMonthsOnARTCalculation.java]: Word the comment appropriately

File: api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/art/MoreThanThreeMonthsOnARTCalculation.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+			Date artStartDate = EmrCalculationUtils.datetimeResultForPatient(artStartDateMap, ptId);
+			if (artStartDate != null && (monthsBetween(artStartDate, context.getNow()) >= 3)){
+				eligible = true;
+			}
+			ret.put(ptId, new BooleanResult(eligible, this));
+		}
+		return ret;
+	}
+
+	int monthsBetween(Date d1, Date d2) {
```
Comment (njorocs) [inline api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/art/MoreThanThreeMonthsOnARTCalculation.java]: looks like a helper method. Can we move it to some common utils class?

**Review by njorocs — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/IsBreastFeedingCalculation.java [hunk 4]
```diff
@@ -42,16 +44,25 @@ public CalculationResultMap evaluate(Collection<Integer> cohort, Map<String, Obj
 
 		Concept ExclusiveBreastFeeding = Dictionary.getConcept(Dictionary.BREASTFED_EXCLUSIVELY);
 		Concept MixedBreastFeeding = Dictionary.getConcept(Dictionary.MIXED_FEEDING);
+		Concept YES = Dictionary.getConcept(Dictionary.YES);
+		Concept BreastFeedingHivFollowupQuestion = Context.getConceptService().getConcept(5632);
```
[the remark this answers is not in the document]
Comment (patryllus) [reply api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/IsBreastFeedingCalculation.java:48]: Done

File: api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/art/MoreThanThreeMonthsOnARTCalculation.java (hunk 2 above)
Comment (patryllus) [reply api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/art/MoreThanThreeMonthsOnARTCalculation.java]: Deleted this file in subsequent PRs

**Pull request closed without merging**